### ISSUE TITLE
[LABELING]: Community Care Service Type label should have placeholder text included for context

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#488]
## Issue
The Community Care service type typeahead uses the `<label>` and `placeholder` to inform users of the type of things they can search for. When the typeahead gains keyboard focus, the placeholder text is removed. This was noted as an SC 3.3.2 violation. Screenshots attached below.

This may require some design input, because we're asking for a longer label to be created. CC'ing @emilywaggoner for awareness.

## Related Issue
https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/489

## Audit Finding
* Note 7, Defect 1 of 2

## Acceptance Criteria
* UPDATE 10/4: As a user, I want the type of services to be added to the label or as descriptive text. Keeping it as a placeholder doesn't line up well with the WCAG guidance. As soon as users start typing, the information disappears.
* As a user, I want the type of things I can search for like "primary care" or "cardiology" to be available anytime. This text should not disappear when the Service Type input receives or loses keyboard focus.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/

## WCAG or Vendor Guidance (optional)
* [Labels or Instructions: Understanding SC 3.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html)
* [G89: Providing expected data format and example](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G89)

## Screenshots or Trace Logs
![Screen Shot 2019-07-05 at 7.45.49 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/552506ce-f335-4069-b357-d9523bef838b)

---
![Screen Shot 2019-07-05 at 7.45.59 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/ea753302-3af9-4c98-8f95-701d94a4593c)

**Comment by kelsonic** [comment]
@1Copenut So I've identified how we can solve this, but this could potentially cause a site-wide change in behavior to our input fields' `placeholder` styling. Currently, we have:
```css
input:focus::-webkit-input-placeholder {
  color: transparent;
}
```

I found this by inspecting the shadow-dom in Chrome's dev tools, which led me to this line in the massive styles.scss file:
![image](https://user-images.githubusercontent.com/12773166/66223966-c5da0180-e6a2-11e9-8e72-0cba07b82fdf.png)

Unfortunately since this is a generated file, I'm having difficulty finding where this style is being applied in the vets-website codebase 🤔 Is it possible that we're inserting global styles that are in a repo different from `vets-website`? @ncksllvn

**Comment by kelsonic** [comment]
[Sent up a PR](https://github.com/department-of-veterans-affairs/vets-website/pull/10852) that fixes **only the `service-type-ahead-input` input field on this page**, but we should discuss whether this should be site-wide since it seems inconsistent.

**Comment by Karak888** [comment]
@kelsonic @1Copenut Please hold off on making any further changes that have to do with Community Cares and Facility Locator overall as we are in discussions with client about making any changes to the Facility Locator overall including any PPMS data (most important). Tickets can move to top of icebox until we are ready to begin any work.  Thanks!

FYSA:  @jenniferlee-dsva

**Comment by kelsonic** [comment]
Closing [the PR](https://github.com/department-of-veterans-affairs/vets-website/pull/10852) for this issue until we want to pick it up again.

**Comment by mmiddaugh** [comment]
I am closing this as a duplicate but will move some of the acceptance criteria into [#3049](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3049) because it addresses a similar/overlapping issue.
